### DISCUSSION TITLE
How i can install Redis for pages?

### DISCUSSION THREAD

**Discussion by AmadeusDeadDmytro** [apostrophecms/apostrophe#2747] [Q&A]
Hello. I need to install Redis to cache pages. 

What is the best way for me to do it?

**Comment by boutell** [comment]
Hi @AmadeusDeadDmytro, what is your goal here? To improve performance beyond the page rendering speed that Apostrophe provides natively? There is no built-in support for caching pages in Redis, but you might be well served by a solution like allowing Cloudflare to aggressively cache your pages, with a separate subdomain you log into without Cloudflare for editing purposes.

I am curious about the traffic levels and/or performance issues that are leading you to pursue such an approach. Although there are certainly high traffic scenarios that might require it, Apostrophe usually renders pages quickly enough that this issue does not come up.
